pkg/mcp/incidents_tool.go

```diff
@@ -17,37 +17,54 @@ import (
 	"github.com/openshift/cluster-health-analyzer/pkg/prom"
 	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
 	"github.com/prometheus/common/model"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/rest"
 )
 
+type IncidentTool struct {
+	mcp.Tool
+	consoleURL string
+}
+
 // IncidentsTool create a new MCP tool for the incidents
-func IncidentsTool() mcp.Tool {
+func IncidentsTool() IncidentTool {
```

Comment:
as per go conventions...

```suggestion
func NewIncidentsTool() IncidentTool {
```